sys/linux/dev_kvm.txt

```diff
@@ -900,3 +922,12 @@ define KVM_ASSIGN_SET_MSIX_ENTRY	0x4010AE74
 define KVM_ASSIGN_SET_MSIX_NR	0x4008AE73
 define KVM_DEASSIGN_DEV_IRQ	0x4040AE75
 define KVM_DEASSIGN_PCI_DEVICE	0x4040AE72
+
+# Register descriptions generated by tools/arm64/registers.go
+kvm_regs_arm64_core = 0x6030000000100000, 0x6030000000100002, 0x6030000000100004, 0x6030000000100006, 0x6030000000100008, 0x603000000010000a, 0x603000000010000c, 0x603000000010000e, 0x6030000000100010, 0x6030000000100012, 0x6030000000100014, 0x6030000000100016, 0x6030000000100018, 0x603000000010001a, 0x603000000010001c, 0x603000000010001e, 0x6030000000100020, 0x6030000000100022, 0x6030000000100024, 0x6030000000100026, 0x6030000000100028, 0x603000000010002a, 0x603000000010002c, 0x603000000010002e, 0x6030000000100030, 0x6030000000100032, 0x6030000000100034, 0x6030000000100036, 0x6030000000100038, 0x603000000010003a, 0x603000000010003c, 0x603000000010003e, 0x6030000000100040, 0x6030000000100042, 0x6030000000100044, 0x6030000000100046, 0x6030000000100048, 0x603000000010004a, 0x603000000010004c, 0x603000000010004e, 0x6030000000100050
```

Comment:
Why do they have this `0x6030` top part? In reg_to_msr we extract only low 16 bits from these, and instructions are 32 bit.
Why does it have 33-bit set? Do we use it in reg_to_msr?

Reply:
> Why do they have this 0x6030 top part?
This is `KVM_REG_ARM64 | KVM_REG_SIZE_U64`, see https://elixir.bootlin.com/linux/v6.10.2/source/include/uapi/linux/kvm.h#L1050

> Why does it have 33-bit set?
Do you mean bit 20? That's KVM_REG_ARM_CORE.

These bits are not used by syzos, but they are used by the KVM_GET_ONE_REG/KVM_SET_ONE_REG ioctls.